.travis.yml

```diff
@@ -3,12 +3,12 @@ node_js:
   - "8"
   - "6"
   - "4"
+before_install:
+  - 'if [ "${TRAVIS_NODE_VERSION}" = "0.6" ]; then npm install -g npm@1.3 ; elif [ "${TRAVIS_NODE_VERSION}" != "0.9" ]; then case "$(npm --version)" in 1.*) npm install -g npm@1.4.28 ;; 2.*) npm install -g npm@2 ;; esac ; fi'
```

Comment:
why do we need this and the `install` script?

Reply:
It ensures npm doesn't break, and deps can be installed on any node version. This will be rebased out tho; only the latest commit on the PR matters.

Reply:
but we dont use versions of node in travis other than 8, 6 and 4, so why test for 0.6 and 0.4, etc?

Reply:
Because then I can copy paste my known-correct Travis config lines from all my other repos :-)